doc/devel/MEP/MEP10.rst

```diff
@@ -44,8 +44,7 @@ these new features.
 Numpy docstring format
 ----------------------
 
-`Numpy docstring format
-<https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt>`_:
+`Numpy docstring format <https://numpydoc.readthedocs.io/en/latest/format.html>`_:
```

Comment:
```suggestion
`Numpy docstring format <https://numpydoc.readthedocs.io/en/stable/format.html>`_:
```
Sorry, I only just noticed this.  "latest" in readthedocs means the development version.  We should link the stable released version.